### PR TITLE
Decimal type for charts and reports

### PR DESCRIPTION
**Pull request by hfazai**

### CODE REVIEW

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt [hunk 1]
```diff
@@ -86,18 +88,18 @@ object ChartSample: Chart() {
...
   init {
     city.add("Tunis") {
-      this[area] = 34600
+      this[area] = decimal("34600")
       this[population] = 1056247
     }
 
     city.add("Kasserine") {
-      this[area] = 806600
+      this[area] = decimal("806600")
```
Comment (h-haddad) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt:96]: what about : decimal(806600.0) to show that we can create decmal from an Double

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt [hunk 2]
```diff
@@ -86,18 +88,18 @@ object ChartSample: Chart() {
...
       this[population] = 439243
     }
 
     city.add("Bizerte") {
       this[population] = 368500
-      this[area] = 568219
+      this[area] = decimal("568219")
```
Comment (h-haddad) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt:102]: what about : decimal(568219L, 0) to test decimal(value: Long, scale: Int)

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/report/ReportTests.kt [hunk 3]
```diff
@@ -233,21 +235,34 @@ object SimpleReport : Report() {
...
       this[name] = "Sofia"
       this[age] = 23
       this[profession] = "Dentist"
+      this[salary] = decimal("2000.55")
     }
     add {
       this[age] = 25
       this[profession] = "Baker"
       this[name] = "Sofia"
+      this[salary] = decimal("2000.55")
```
Comment (h-haddad) [inline galite-tests/src/test/kotlin/org/kopi/galite/tests/report/ReportTests.kt:265]: can u use other constructor for decimal with double for example

**Review by h-haddad — CHANGES_REQUESTED** [review]

File: galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt (hunk 1 above)
Comment (hfazai) [reply galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt:96]: The recommanded way is to use string, floats are not precise

**Review by h-haddad — APPROVED** [review]

**Pull request merged**